src/stores/connectors/keys/update.go

```diff
@@ -0,0 +1,38 @@
+package keys
+
+import (
+	"context"
+	"github.com/consensys/quorum-key-manager/src/stores/store/database"
+	"github.com/consensys/quorum-key-manager/src/stores/store/entities"
+)
+
+func (c Connector) Update(ctx context.Context, id string, attr *entities.Attributes) (*entities.Key, error) {
+	logger := c.logger.With("id", id)
+	logger.Debug("updating key")
+
+	key, err := c.db.Keys().Get(ctx, id)
+	if err != nil {
+		return nil, err
+	}
+	key.Tags = attr.Tags
+
+	err = c.db.RunInTransaction(ctx, func(dbtx database.Database) error {
+		derr := c.db.Keys().Update(ctx, key)
```

Comment:
is go_pg automatically updating the field `updated_at` or should you take care of it?

Reply:
adding tests

Reply:
tests added